index.html

```diff
@@ -0,0 +1,205 @@
+          </div>
+          <div class="tech-topics-card">
+            <img class="tech-logo" src="img/js.png" alt="js-logo">
+            <h3 class="fw-bold fs-500 text-accent-400">JavaScript</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="tech-topics-card">
+            <img  class="tech-logo" src="img/java.png" alt="java-logo">
+            <h3 class="fw-bold fs-500 text-accent-400">Java</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+          <a href="#" class="see-more">Check Out Our Dev Topics</a>
+          <div class="btn-wrap">
+          <button class="see-more-btn">Dev Topics</button>
+          </div>
+      </div>
+    </section>
+    
+      <!-- tech topics section ends here -->
+
+      <!-- contributors section starts here -->
+
+    <section id="contributors">
+      <div class="container ">
+        <div>
+          <h2 class="title fs-700 text-neutral-900 fw-semi-bold">Top contributors</h2>
+          <div class="indicator"></div>
+        </div>
+        <div class="contributors-container">
+        </div>
+      </div>
+    </section>
+
+      <!-- contributors section ends here -->
+
+      <!-- main sponsors section starts here -->
+
+    <section > 
+      <div id="main-sponsors">
+      <div class="container">
+          <div>
+            <h2 class="title fs-700 text-accent-100: #F8F8F8; fw-semi-bold">Main Sponsors</h2>
+            <div class="indicator"></div>
+          </div>
+          <div class="sponsors-icon-container">
+            <ul  class="sponsors-logo-list" aria-label="sponsors logo">
+              <li><img src="img/box-logo.svg" alt="sponsors-logo"></li>
+              <li><img src="img/eventbrite.svg"alt="sponsors-logo"></li>
+              <li><img src="img/nasdaq.svg"alt="sponsors-logo"></li>
+              <li><img src="img/netapp.svg"alt="sponsors-logo"></li>
+              <li><img src="img/tcstata.svg"alt="sponsors-logo"></li>
+            </ul>
+          </div>
+      </div>
+      </div>
+    </section>
+  </main>
+  
+      <!-- main sponsors section ends here -->
+
+      <!-- footer section starts here -->
+
+  <footer>
+    <div class="container">
+      <div class="footer-container">
+        <div class="newslogo">
+          <a href="index.html"><img src="img/newslogo.png" alt="logo"></a>
+        </div>
+        <div>
+            <p>Copyright 2022. All Rights Reserved</p>
+            <div class="lorem">
+            <p> Lorem ipsum, dolor sit amet consectetur adipisicing elit. Architecto<br> at inventore maxime vel qui, commodi et! Eveniet dolorem saepe consectetur.</p>
+            </div>
+        </div>
+      </div>
+    </div>
+  </footer>
+<script src="dynamiccards.js"></script>
+<script src="./menu-nav.js"></script>
```

Comment:
- [ ] Your code is breaking on footer and header.

![footer](https://user-images.githubusercontent.com/65860678/198829385-3a3a4598-9bd6-4bb5-8984-8e7e7d1642af.png)

Reply:
the code you see next to the logo is not broken code, it was a bad design error because I did not think that it would look like broken code when I made it apart of my logo - my website is called "div news" so i thought i would be creative and write my logo title "div news" as a piece of code "<div id="News>" so i called the reviewer and cleared it up with him by showing him my logo as he could not find that line of code anywhere in my code besides the logo